examples/reg.py

```diff
@@ -413,8 +425,10 @@ def __strip_root_key(self, dce, keyName):
             raise Exception('Error parsing keyName %s' % keyName)
         if rootKey.upper() == 'HKLM':
             ans = rrp.hOpenLocalMachine(dce)
-        elif rootKey.upper() == 'HKU':
+        elif rootKey.upper() == 'HKCU':
```

Comment:
nice to have this fixed. please update help in line 551, as it's referencing root key names, and it's not currently contemplating HKCU